repository/src/test/java/org/alfresco/encryption/EncryptionTests.java

```diff
@@ -101,6 +101,18 @@
 
 	private UserTransaction tx;
 
+	static SecureRandom random;
+	static {

```

Comment:
Consider moving initialisation to a static method. It will allow the field to be `final`.